Dockerfile.windows

```diff
@@ -128,55 +161,120 @@ SHELL ["powershell", "-command"]
 #  - GO_VERSION must be consistent with 'Dockerfile' used by Linux.
 #  - FROM_DOCKERFILE is used for detection of building within a container.
 ENV GO_VERSION=1.7.3 `
-    GIT_LOCATION=https://github.com/git-for-windows/git/releases/download/v2.10.1.windows.1/Git-2.10.1-64-bit.exe `
+    GIT_VERSION=2.10.2 `
```

Comment:
But you have to build from source code - that's a development branch, not release. Release is still at 2.10.2